org.eclipse.scanning.test/src/org/eclipse/scanning/test/event/queues/dummy/DummyBeanProcess.java

```diff
@@ -27,5 +27,20 @@ public DummyBeanProcess(T bean, IPublisher<T> publisher, Boolean blocking) throw
 	public Class<DummyBean> getBeanClass() {
 		return DummyBean.class;
 	}
+	
```

Comment:
You could put this once in the original interface:

 +	default void postMatchCompleted() throws EventException {
 +		//Implement as required
 +	}
 +
 +	default void postMatchTerminated() throws EventException {
 +		//Implement as required
 +	}
 +
 +	default void postMatchFailed() throws EventException {
 +		//Implement as required
 +	}